java-api/src/main/java/com/sap/cloud/security/config/FileSystemAccessor.java

```diff
@@ -0,0 +1,39 @@
+package com.sap.cloud.security.config;
+
+import javax.annotation.Nullable;
+import java.io.File;
+import java.util.Properties;
+
+/**
+ * The interface for K8s File system access.
+ */
+public interface FileSystemAccessor {
+
+
+    /**
+     * Gets all bound Xsuaa service instances
+     *
+     * @return the array of files
+     */
+    @Nullable
+    File[] getXsuaaBindings();
```

Comment:
I wouldn't expos that... i would only expose two methods:
- getXsuaaServiceProperties()
- getIasServiceProperties()